RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -75,7 +77,8 @@ struct PurchaseInformation {
          isTrial: Bool,
          isCancelled: Bool,
          latestPurchaseDate: Date?,
-         customerInfoRequestedDate: Date
+         customerInfoRequestedDate: Date,
+         managePurchaseURL: URL?
```

Comment:
maybe `managementUrl` for consistency

Reply:
check https://github.com/RevenueCat/purchases-ios/pull/5050#discussion_r2076280845

Reply:
then `managementURL` instead of `managePurchaseURL`?